packages/monaco/src/browser/monaco-editor-provider.ts

```diff
@@ -190,6 +191,24 @@ export class MonacoEditorProvider {
         return editor;
     }
 
+    protected addDynamicKeybindings(editor: MonacoEditor, standaloneCommandService: monaco.services.StandaloneCommandService, scope: KeybindingScope): void {
+        const keybindings = this.keybindingRegistry.getKeybindingsByScope(scope);
+        if (keybindings.length > 0) {
+            const keybindingService = editor.getControl()._standaloneKeybindingService;
+            if (keybindingService) {
+                keybindingService._commandService = standaloneCommandService;
```

Comment:
I want to put attention to this line. 
It was added in order to be able to run addDynamicKeybinding where vscode has a check concerning it [there](https://github.com/microsoft/vscode/blob/c47d83b293181d9be64f27ff093689e8e7aed054/src/vs/editor/standalone/browser/simpleServices.ts#L344):
Not sure that is best solution but I couldn't find an alternative.
![image](https://user-images.githubusercontent.com/71069170/100202298-cd451580-2f09-11eb-92ec-00d95b4cd313.png)